File: baekjoon/Level/Silver/13417. 카드 문자열/카드 문자열.swift
let T = Int(readLine()!)!

for _ in 0..<T {
    let N = Int(readLine()!)!
    let alphabet = readLine()!.split(separator: " ")
    var result: [String] = []
    for al in alphabet {
        let a = String(al)
        if result.isEmpty {
            result.append(a)
            continue
        }
        if result.first! >= a {
            result.insert(a, at: 0)
        } else {
            result.append(a)
        }
    }
    print(result.joined())
}

File: baekjoon/Level/Silver/13417. 카드 문자열/카드 문자열.py
T = int(input())

for _ in range(T):
    N = int(input())
    alphabet = input().split(" ")
    result = []
    for a in alphabet:
        if len(result) == 0:
            result.append(a)
            continue
        if result[0] >= a:
            result.insert(0, a)
        else:
            result.append(a)
    print("".join(result))